packages/flutter_tools/lib/src/base/utils.dart

```diff
@@ -59,7 +59,7 @@ String sentenceCase(String str, [String? locale]) {
   if (str.isEmpty) {
     return str;
   }
-  return toBeginningOfSentenceCase(str, locale)!;
+  return ArgumentError.checkNotNull(toBeginningOfSentenceCase(str, locale));
```

Comment:
can you put a TODO here to remove this once `toBeginningOfSentenceCase` is non-nullable?

Reply:
I am not willing to put my name into the TODO and not sure if it will let me do a different format of a TODO.

Reply:
you can put `TODO(christopherfujino)`

Reply:
```suggestion
  // TODO(christopherfujino): Remove this check after the next release of intl
  return ArgumentError.checkNotNull(toBeginningOfSentenceCase(str, locale));
```

Reply:
Done